tests/ui/specs/VirtualList/VirtualGridList/qa-VirtualGridList/VirtualGridList-GT-28987-specs.js

```diff
@@ -0,0 +1,35 @@
+const Page = require('../VirtualGridListPage');
+const {expectFocusedItem} = require('../../VirtualList-utils');
+
+describe('qa-VirtualGridList', function () {
+	beforeEach(function () {
+		Page.open('', '?locale=ar-SA');
+	});
+
+	it('should spotlight move on last item [GT-28987]', function () {
+		// Step 2-1: Select 'Horizontal'
+		Page.buttonDirectionChange.moveTo();
+		Page.spotlightSelect();
+
+		// Step 3-1: 5-way spot item 0.
+		Page.item(0).moveTo();
+		Page.spotlightRight();
+		expectFocusedItem(0);
+		// Step 3-2: 5-way Left to Item 16.
+		for (let i = 0; i < 8; i++) {
+			Page.spotlightLeft();
+			Page.delay(200);
+		}
+		Page.delay(500);
+		// Step 3-2 Verify: Spotlight is on item 16.
+		expectFocusedItem(16);
+		const curScrollThumbPosition = Page.scrollThumbPosition();
+
```

Comment:
How about verifying "curScrollThumbPosition > 0" or "curScrollThumbPosition != initialThumbPoisition"

Reply:
It compared scrollthumb's positon on item16 and item14. This value is not 0.
When press 5-way right on item 16, the scrollthumb's position should not change.